test/client/spec/helpers/constrain-tick-labels.spec.js

```diff
@@ -0,0 +1,433 @@
+/* eslint no-unused-expressions: 0 */
+/* eslint-disable no-magic-numbers */
+require("babel-polyfill");
+import constrainTickLabels from "src/components/victory-axis/constrain-tick-labels";
+
+const mockMeasureText = function * (measurements) {
+  const mapper = (measurement) => (tick, index) => {
+    return {
+      text: tick,
+      measurement: measurement[index],
+      index
+    };
+  };
+  for (const measurement of measurements) {
+    const tickValues = yield;
+    yield tickValues.map(mapper(measurement));
+  }
+};
+
+const mockMeasureTicks = (generator) => {
+  return (props) => {
+    const result = generator.next(props.tickValues).value;
+    generator.next();
```

Comment:
Hmm, explain why we're assigning a `const` from the first call to `next().value`, and then calling `next()` without caring about the result?